versioned_docs/version-1.21.5/concepts/events.md

```diff
@@ -82,8 +82,19 @@ public class YourMod {
 
 ### `@EventBusSubscriber`
 
+<Tabs defaultValue="latest">
```

Comment:
Do we really want this? We generally don't use per-Neo-version docs in other places, so why here?

Reply:
Ah, I didn't realize that 1.21.5 was already stable. Alright then.